SoojungKim/baekjoon/BOJ_10172.java

```diff
@@ -0,0 +1,11 @@
+package baekjoon;
+
+public class BOJ_10172 {
+    public static void main(String[] args) {
+        System.out.println("|\\_/|");
+        System.out.println("|q p|   /}");
+        System.out.println("( 0 )\"\"\"\\");
+        System.out.println("|\"^\"`    |");
+        System.out.println("||_/=\\\\__|");
```

Comment:
```suggestion
        StringBuilder sb = new StringBuilder();
        sb.append("|\\_/|\n")
            중략
            .append("||_/=\\\\__|");
        System.out.println(sb);
```

`StringBuilder`를 사용했으면 더 좋았을 것 같아요